src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -81,6 +107,48 @@
  *       value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void equals() { // violation, should be annotated with &#64;Override
+ *
+ *     }
+ * }
+ *
+ * interface Test {
+ *
+ *      &#47;** {&#64;inheritDoc} *&#47;
+ *      public abstract void test(); // violation, should be annotated with &#64;Override
```

Comment:
`public abstract` please remove these redundant modifiers, it is public and abstract by default

Reply:
I am done